[2024-03-13T17:13:04.882+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion.load_data manual__2024-03-13T17:12:58.948667+00:00 [queued]>
[2024-03-13T17:13:04.885+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion.load_data manual__2024-03-13T17:12:58.948667+00:00 [queued]>
[2024-03-13T17:13:04.886+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-03-13T17:13:04.891+0000] {taskinstance.py:1382} INFO - Executing <Task(_PythonDecoratedOperator): load_data> on 2024-03-13 17:12:58.948667+00:00
[2024-03-13T17:13:04.894+0000] {standard_task_runner.py:57} INFO - Started process 1698 to run task
[2024-03-13T17:13:04.896+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion', 'load_data', 'manual__2024-03-13T17:12:58.948667+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/fetch-api-data.py', '--cfg-path', '/tmp/tmpshbclzkw']
[2024-03-13T17:13:04.898+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask load_data
[2024-03-13T17:13:04.917+0000] {task_command.py:416} INFO - Running <TaskInstance: data_ingestion.load_data manual__2024-03-13T17:12:58.948667+00:00 [running]> on host 5903490e815f
[2024-03-13T17:13:04.980+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-03-13T17:12:58.948667+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-13T17:12:58.948667+00:00'
[2024-03-13T17:13:04.983+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-03-13T17:13:04.986+0000] {logging_mixin.py:154} INFO - Table created successfully or already exists
[2024-03-13T17:13:04.990+0000] {logging_mixin.py:154} INFO - Data inserted successfully
[2024-03-13T17:13:04.990+0000] {logging_mixin.py:154} INFO - PostgreSQL connection closed
[2024-03-13T17:13:04.990+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-03-13T17:13:04.993+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_ingestion, task_id=load_data, execution_date=20240313T171258, start_date=20240313T171304, end_date=20240313T171304
[2024-03-13T17:13:05.036+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-03-13T17:13:05.044+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
